owncloudApp/src/main/java/com/owncloud/android/providers/DocumentsStorageProvider.kt

```diff
@@ -206,17 +206,24 @@ class DocumentsStorageProvider : DocumentsProvider() {
         updateCurrentStorageManagerIfNeeded(docId)
 
         return FileCursor(projection).apply {
-            val ocFile: OCFile? = currentStorageManager?.getFileById(docId)
-            if (ocFile != null) addFile(ocFile)
+            getFileById(docId)?.let { addFile(it) }
         }
     }
 
     override fun onCreate(): Boolean = true
 
     override fun queryRoots(projection: Array<String>?): Cursor {
-        initiateStorageMap()
         val result = RootCursor(projection)
         val contextApp = context ?: return result
+        // If OwnCloud is protected with passcode or pattern, return empty cursor.
+        val preferences = PreferenceManager.getDefaultSharedPreferences(context).apply { }
```

Comment:
You wrote here an apply with an empty body, is there any reason or did you just forget it?

Reply:
I forgot it..